src/activities/hangman/hangman.js

```diff
@@ -136,6 +137,9 @@ function processKeyPress(text) {
     // Add the character to the already typed characters
     alreadyTypedLetters.push(text);
 
+    // add
+    items.guessedText.text=items.guessedText.text.concat(text, ", ");
```

Comment:
In the recent commit, I solved it in a similar way in the _form()_ method

In that method, I initialized a string with "Attempted: ". Then I sorted the "alreadyTypedLetters" 
array and inserted it into the forming string one by one. When the full traversal is over, I am 
returning the string after passing it through qsTr(), and that is shown in the top of the activity.